### ISSUE TITLE
[Broken Website] att.com

### ISSUE THREAD

**Issue by ghost** [darkreader/darkreader#10487]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Top White Bar

### Website Address

https://www.att.com/

### Steps To Reproduce

Visit the website with Dark Reader enabled.
Notice the top white bar in the screenshot needs fixing.

### Screenshots

![att](https://user-images.githubusercontent.com/95879668/207650588-b00b49a7-f147-4cab-974c-d33341c2af95.png)

### Operating System

Arch Linux

### Web Browser name and version

Firefox 108

### Dark Reader version

4.9.60

### Additional Context

I went through my older issues and found #9025, but most elements I had reported there have been fixed or are no longer applicable. A slight redesign to the website made the top white bar larger, and now it includes more elements. I decided to close the old report and create a new one due to the changes made to the website.

Also, Gusted had a note in the old report that stated the following:

> Note for later: Apparently constructed stylesheets are "broken" on this site.

**Comment by Gusted** [comment]
Yeah, it's indeed and still broken for some reason... `.adoptedStylesheets` must return a array and not a vague `opaque` object... Still have zero clue.
![image](https://user-images.githubusercontent.com/25481501/210026438-e6984a55-96bb-400c-9b53-23da4d0c38ac.png)
